backend/shared/shared/azure_adfs/auth.py

```diff
@@ -12,6 +12,27 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def adfs_login_group_name():
+    """
+    Name of the user group that signifies that the user has logged in using ADFS i.e.
+    HelsinkiAdfsAuthCodeBackend
+
+    :return: Value of settings.ADFS_LOGIN_GROUP_NAME if it exists, otherwise
+             "ADFS login"
+    """
+    return getattr(django_settings, "ADFS_LOGIN_GROUP_NAME", "ADFS login")
+
+
+def is_adfs_login(user) -> bool:
```

Comment:
`was_logged_in_using_adfs` would be better name because user is not necessary active anymore

Reply:
It was concluded after discussion that `is_adfs_login` is ok as e.g. `is_authenticated` is used by Django itself rather than e.g. `was_authenticated`. So, `is_adfs_login` is in line with the precedent set by Django.